libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -87,6 +87,13 @@ export class CounterfactualList extends React.Component<
   public render(): React.ReactNode {
     const items = this.getItems();
     const columns = this.getColumns();
+    const targetFeature = this.getTargetFeatureName();
```

Comment:
this new section seems belong to get columns, could be better move to getColumns function.

Reply:
Done